src/js/utils/Select.jsx

```diff
@@ -107,9 +107,16 @@ class Select extends Component {
       multi, placeholder, showLabelTooltip, value, defaultPlaceholder,
     } = this.props;
 
+    if (value.translatedName) {
+      return (
+        <div className="p-1">
+          {value.name}
+        </div>);
+    }
+
     if (showLabelTooltip) {
       const valueMapped = multi && value ? this.props.value.map(v => v && v.label) : [];
-      const valueLabel = multi ? valueMapped.join(', ') : (value.label || value.name);
+      const valueLabel = multi ? valueMapped.join(', ') : value.label || value.name;
```

Comment:
are you sure those `( )` are not needed here? Nothing will break due to lack of it? Is there a reason you deleted them?